aliases/available/docker.aliases.bash

```diff
@@ -0,0 +1,16 @@
+cite 'about-alias'
+about-alias 'docker abbreviations'
+
+alias dklc='docker ps -l'  # List last Docker container
+alias dklcid='docker ps -l -q'  # List last Docker container ID
+alias dklcip="docker inspect `dklcid` | grep IPAddress | cut -d '\"' -f 4"  # Get IP of last Docker container
+alias dkps='docker ps'  # List running Docker containers
+alias dkpsa='docker ps -a'  # List all Docker containers
+alias dki='docker images'  # List Docker images
+alias dkrmac='docker rm $(docker ps -a -q)'  # Delete all Docker containers
+alias dkrmlc='docker-remove-most-recent-container'  # Delete most recent (i.e., last) Docker container
```

Comment:
I see there is a larger issue to decide here for the project so, in the meantime, I just went ahead and converted my added underscore functions to use dashes like the pre-existing ones in the docker functions file. They can be cleaned up with others in the project if underscores are the final decision. Let me know if there is anything else that needs to be cleaned up with this PR. Thanks!